cmd/regula/cli/cli.go

```diff
@@ -32,22 +34,50 @@ type Config struct {
 }
 
 // LoadConfig loads the configuration from the environment or command line flags.
-func LoadConfig() (*Config, error) {
+// The args hold the command line arguments, as found in os.Argv.
+// It returns flag.ErrHelp if the -help flag is specified on the command line.
+func LoadConfig(args []string) (*Config, error) {
 	var cfg Config
-	cfg.LogLevel = zerolog.DebugLevel.String()
+	flag := stdflag.NewFlagSet("", stdflag.ContinueOnError)
+	flag.StringVar(&cfg.Etcd.Namespace, "etcd-namespace", "", "etc namespace to use")
+	flag.StringVar(&cfg.LogLevel, "log-level", zerolog.DebugLevel.String(), "debug level")
 	cfg.Etcd.Endpoints = []string{"127.0.0.1:2379"}
-	cfg.Server.Address = "0.0.0.0:5331"
-	cfg.Server.Timeout = 5 * time.Second
-	cfg.Server.WatchTimeout = 30 * time.Second
+	flag.Var(commaSeparatedFlag{&cfg.Etcd.Endpoints}, "etcd-endpoints", "comma separated etc endpoints")
+	flag.StringVar(&cfg.Server.Address, "addr", "0.0.0.0:5331", "server address to listen on")
+	flag.DurationVar(&cfg.Server.Timeout, "server-timeout", 5*time.Second, "server timeout (TODO)")
+	flag.DurationVar(&cfg.Server.WatchTimeout, "server-watch-timeout", 30*time.Second, "server watch timeout (TODO)")
 
-	err := confita.NewLoader(env.NewBackend(), flags.NewBackend()).Load(context.Background(), &cfg)
+	err := confita.NewLoader(env.NewBackend()).Load(context.Background(), &cfg)
 	if err != nil {
 		return nil, err
 	}
-
+	if err := flag.Parse(os.Args[1:]); err != nil {
```

Comment:
```suggestion
	if err := flag.Parse(args[1:]); err != nil {
```

Let's use the args received in parameters to ease testing.